mlflow/transformers/model_io.py

```diff
@@ -144,7 +144,7 @@ def _load_model(model_name_or_path, flavor_conf, accelerate_conf, device, revisi
 
     load_kwargs["device"] = device
     if torch_dtype := flavor_conf.get(FlavorKey.TORCH_DTYPE):
-        load_kwargs[FlavorKey.TORCH_DTYPE] = _deserialize_torch_dtype(torch_dtype)
```

Comment:
Now `flavor_conf` should always have `torch.dtype` so we don't need deserialization here.